library/wizard/test/wizard_test.rb

```diff
@@ -0,0 +1,62 @@
+#! /usr/bin/env rspec
```

Comment:
I guess we need to include here the missing copyright comments, isn't it?

https://github.com/yast/yast-yast2/blob/master/library/systemd/test/yast2/compound_service_test.rb#L4-L21

Reply:
:+1: